src/AllCocktailsPage/AllCocktailsPage.tsx

```diff
@@ -1,21 +1,22 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useRef } from "react";
 import { AllCocktailsDetails } from '../App/App';
-import { getAllCocktails } from "../apiCalls";
 import CocktailCard from '../CocktailCard/CocktailCard';
 import './AllCocktailsPage.scss';
 
 export interface AllCocktailsProps {
-  allCocktails: AllCocktailsDetails[];
+	givenCocktails: AllCocktailsDetails[];
```

Comment:
more semantic name since this component will handle displaying ALL cocktails, or FAVORITE cocktails, or LOGGED cocktails (soon)

Reply:
I like it!